src/app/pages/organizations/developers/developer/join.jsx

```diff
@@ -93,94 +132,108 @@ function ChplJoinDevelopers({ id }) {
-                    <TableCell>{developer.name}</TableCell>
-                    <TableCell>{developer.status.status}</TableCell>
-                    <TableCell>
-                      <Button
-                        onClick={() => addDeveloper(undefined, developer)}
-                        disabled={!canAdd(developer)}
-                      >
-                        Add
-                      </Button>
-                    </TableCell>
-                  </TableRow>
-                ))}
-              </TableBody>
-            </Table>
-          </TableContainer>
-        </CardContent>
-      </Card>
+      <Box p={8} bgcolor="#fff">
+        <Typography variant="h1">
+          Join Developers
+        </Typography>
+      </Box>
+      <Container disableGutters maxWidth="xl">
+        <Box className={classes.boxContainer}>
+          <Card className={classes.stickyCardContainer}>
+            <CardHeader title={`Developers joining ${activeDeveloper.name}`} />
+            <CardContent>
+              {developersToJoin.length === 0 && 'None selected'}
+              {developersToJoin.length > 0
+                && (
+                  <TableContainer>
+                    <Table>
+                      <TableHead>
+                        <TableRow>
+                          <TableCell>Code</TableCell>
+                          <TableCell>Name</TableCell>
+                          <TableCell>Status</TableCell>
+                          <TableCell><span className="sr-only">Action</span></TableCell>
+                        </TableRow>
+                      </TableHead>
+                      <TableBody>
+                        {developersToJoin.map((developer) => (
+                          <TableRow key={developer.id}>
+                            <TableCell>{developer.developerCode}</TableCell>
+                            <TableCell>{developer.name}</TableCell>
+                            <TableCell>{developer.status.status}</TableCell>
+                            <TableCell>
+                              <Button
+                                onClick={() => removeDeveloper(developer)}
+                                variant="outlined"
+                                className={classes.errorColor}
+                                endIcon={<ClearIcon fontSize='small' />}
+                              >
+                                Remove
+                              </Button>
+                            </TableCell>
+                          </TableRow>
+                        ))}
+                      </TableBody>
+                    </Table>
+                  </TableContainer>
+                )}
+            </CardContent>
+          </Card>
+          <Card className={classes.cardContainer}>
+            <CardHeader title={`Select Developers joining ${activeDeveloper.name}`} />
+            <CardContent>
+              { /* eslint-disable react/jsx-props-no-spreading */}
```

Comment:
This comment should be one line lower. When we disable an eslint rule, we want to do it in as small a place as possible, and right now this comment doesn't line up, in indention, with the "re-enable" line 199